src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -400,7 +394,11 @@ private class AppTaskLauncherActor(
       case _ => "not backing off"
     }
 
-    s"$tasksToLaunch tasksToLaunch, ${inFlightTaskLaunches.size} in flight. $backoffStr"
+    val inFlight = inFlightTaskLaunches.size
+    val requested = tasksMap.size - inFlight
+    val matchInstanceCount = tasksMap.size + tasksToLaunch - app.instances
+    val matchInstanceStr = if (matchInstanceCount == 0) "" else s"instance count discrepancy $matchInstanceCount."
```

Comment:
isn't "delta" more adequate than discrepancy?

discrepancy: an illogical or surprising lack of compatibility or similarity between two or more facts.

The name of the variables is also a bit confusing. Shouldn't it be instanceCountDelta/Difference/Gap?